src/tests/neighbourhood.ts

```diff
@@ -33,8 +34,12 @@ export default function neighbourhoodTests(testContext: TestContext) {
                 //Check that we got an ad4m url back
                 expect(publishPerspective.split("://").length).toBe(2);
 
+                const perspective = await ad4mClient.perspective.byUUID(create.uuid);
+                expect(perspective?.neighbourhood).toBeDefined();
+
                 const join = await ad4mClient.neighbourhood.joinFromUrl(publishPerspective );
                 expect(join.sharedUrl).toBe(publishPerspective);
+                expect(join.neighbourhood).toBeDefined();
```

Comment:
Should we also check the neighbourhood's fields (`linkLanguage` and `meta`)?

Reply:
Sure adding and then merging